app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -546,6 +546,8 @@ class GleanMetricsService(private val context: Context) : MetricsService {
             adjustAdGroup.set(context.settings().adjustAdGroup)
             adjustCreative.set(context.settings().adjustCreative)
             adjustNetwork.set(context.settings().adjustNetwork)
+            hasTopSites.set(context.settings().topSitesSize > 0)
+            topSitesCount.add(context.settings().topSitesSize)
```

Comment:
so I think we don't want to use a coutner here, this will add the topSitesSize to the counter on every launch